content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,490 @@
+---
+title: "COVID-19 RDT Reference Application"
+linkTitle: "COVID-19 RDT"
```

Comment:
Looking at the examples we have it may make sense to distinguish this as a "testing" workflow

```suggestion
title: "Reference Application using RDT for COVID-19 Testing"
linkTitle: "COVID-19 Testing"
```